README.md

```diff
@@ -39,7 +39,7 @@ This brief tutorial shows **where some of the most used and quoted sysctl/networ
 1. Driver will `schedule a NAPI`, clear the `hard IRQ` and return
 1. Driver raise a `soft IRQ (NET_RX_SOFTIRQ)`
 1. NAPI will poll data from the receive ring buffer until `netdev_budget_usecs` timeout or `netdev_budget` and `dev_weight` packets
-1. Linux will also allocated memory to `sk_buff`
+1. Linux will also allocates memory to `sk_buff`
```

Comment:
Still incorrect grammar: 

Correct would be: 

> Linux will also allocate memory to `sk_buff`.

Reply:
`s/allocates/allocate`

Reply:
true that, fixed !